### PR TITLE
LUIS Connection

### PR DESCRIPTION
**Pull request by AdrianMachado**
LUIS connection established, basic conversation example added.

### CODE REVIEW

**Review by jack-mo — APPROVED** [review]

**Review by jack-mo — APPROVED** [review]
LGTM 👍

File: app.js [hunk 1]
```diff
@@ -25,13 +25,25 @@ server.post('/api/messages', connector.listen());
 
 // Static root page
 server.get('/', (req, res) => {
-    res.send(200, `API Version ${config.apiVersion} currently running.`);
+    res.send(200, 'API Version ${config.apiVersion} currently running.');
 });
 
+// LUIS Setup
+var model = 'https://api.projectoxford.ai/luis/v1/application?id=6bfcb96e-0dbb-443d-8c75-36224da1b80f&subscription-key=' + config.LuisSubscriptionKey;
```
Comment (timmui) [inline app.js:32]: Put id in config var

Comment (timmui) [reply app.js:32]: Also, use ES6 string formatting
See line 28

File: config.js [hunk 2]
```diff
@@ -13,4 +13,5 @@ module.exports = {
     MSAppId: process.env.MICROSOFT_APP_ID,
     MSAppPassword: process.env.MICROSOFT_APP_PASSWORD,
     MSTextAnalyticsKey: process.env.MICROSOFT_TEXT_ANALYTICS_KEY,
+    LuisSubscriptionKey: process.env.LUIS_SUBSCRIPTION_KEY,
```
Comment (timmui) [inline config.js:16]: prepend `MICROSOFT_` to the key

**Review by timmui — CHANGES_REQUESTED** [review]
Fix shit mentioned above

**Pull request merged**